pypeit/spectrographs/keck_mosfire.py

```diff
@@ -171,14 +171,56 @@ def config_specific_par(self, scifile, inp_par=None):
             par['reduce']['slitmask']['assign_obj'] = True
             # force extraction of undetected objects
             par['reduce']['slitmask']['extract_missing_objs'] = True
-            if 'long2pos' in self.get_meta_value(headarr, 'decker'):
+            if 'long2pos' in decker:
                 # exclude the random slits outside the long2pos from slit tracing
                 pix_start, pix_end = self._long2pos_pos()
                 par['calibrations']['slitedges']['exclude_regions'] = ['1:0:{}'.format(pix_start),
                                                                        '1:{}:2040'.format(pix_end)]
                 # assume that the main target is always detected, i.e., skipping force extraction
                 par['reduce']['slitmask']['extract_missing_objs'] = False
 
+        # wavelength calibration
```

Comment:
Is there a reason why you didn't just create one OH line list for all of the MOSFIRE setups? I think there is no difference except less code, and possibly less chance for confusion from lines not within the filter, which for full_template should basically never happen. The main upside is that if there were another IR instrument with comparable resolution to MOSFIRE, we could use that list without re-running the wave model code again.